src/plugins/intel_cpu/src/nodes/reference.cpp

```diff
@@ -94,7 +88,13 @@ void Reference::executeDynamicImpl(dnnl::stream strm) {
             if (memory->getSize() != tensor.get_byte_size()) {
                 THROW_CPU_NODE_ERR("output tensor data size mismatch occurred during the inference on output port number ", i);
             }
-            cpu_memcpy(memory->getData(), tensor.data(), tensor.get_byte_size());
+            if (tensor.get_element_type() == element::string) {
```

Comment:
Such kind of code is duplicated throw all the plugin. 
@maxnick  @nshchego  Does it makes sense to inroduce new helper function (like cpu_copy) for different combinations of Memory/Tensors?

Reply:
Actually we have `IMemory::load` method for that purpose. Probably it's just enough to optimize it by replacing standard memcpy with cpu_parallel_memcpy in the load implementation. Whenever we need to transfer data from one memory object into another, we simply call `load`.